core/tee/tee_svc_storage.c

```diff
@@ -834,22 +834,29 @@ TEE_Result tee_svc_storage_alloc_enum(uint32_t *obj_enum)
 	struct tee_ta_session *sess;
 	TEE_Result res;
 
-	if (obj_enum == NULL)
-		return TEE_ERROR_BAD_PARAMETERS;
+	if (!obj_enum) {
+		res = TEE_ERROR_BAD_PARAMETERS;
+		goto exit;
```

Comment:
I don't see the point with adding an `exit:` label for this function, same goes for all the other functions below that just does a `return res`.